app/assets/stylesheets/pages/_home.scss

```diff
@@ -11,3 +11,7 @@ $offset: 16%;
     margin-left: $offset;
   }
 }
+
+.i18n-trans-link {
```

Comment:
(that will do the same thing: apply a margin-left (ml) of 1 spacing unit which is .5rem)